oap-server/analyzer/agent-analyzer/pom.xml

```diff
@@ -39,8 +39,9 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.codehaus.groovy</groupId>
-            <artifactId>groovy</artifactId>
+            <groupId>org.apache.skywalking</groupId>
```

Comment:
this is the only place where groovy is used? if so, you need to update/remove the license and the dependency in the `oap-server/pom.xml`